CMakeLists.txt

```diff
@@ -94,10 +94,13 @@ execute_process(
 )
```

Comment:
It seems not necessary to compute `OT_VERSION` when `OT_PACKAGE_VERSION` is defined? Should we remove guard this expression?

Reply:
Fixed! Thanks.